src/rpc/channel.rs

```diff
@@ -235,25 +237,28 @@ where
         &self,
         params: ListChannelsParams,
     ) -> Result<ListChannelsResult, ErrorObjectOwned> {
-        let channels = self.store.get_channel_states(params.peer_id);
-        Ok(ListChannelsResult {
-            channels: channels
-                .into_iter()
-                .filter_map(|(peer_id, channel_id, _state)| {
-                    self.store
-                        .get_channel_actor_state(&channel_id)
-                        .map(|state| Channel {
-                            channel_id,
-                            peer_id,
-                            state: state.state,
-                            local_balance: state.get_local_balance(),
-                            remote_balance: state.get_remote_balance(),
-                            sent_tlc_balance: state.get_sent_tlc_balance(),
-                            received_tlc_balance: state.get_received_tlc_balance(),
-                        })
-                })
-                .collect(),
-        })
+        let mut channels: Vec<_> = self
+            .store
+            .get_channel_states(params.peer_id)
+            .into_iter()
+            .filter_map(|(peer_id, channel_id, _state)| {
+                self.store
+                    .get_channel_actor_state(&channel_id)
+                    .map(|state| Channel {
+                        channel_id,
+                        peer_id,
+                        state: state.state,
+                        local_balance: state.get_local_balance(),
+                        remote_balance: state.get_remote_balance(),
+                        sent_tlc_balance: state.get_sent_tlc_balance(),
+                        received_tlc_balance: state.get_received_tlc_balance(),
+                        created_at: state.created_at,
+                    })
+            })
+            .collect();
+        // Sort by created_at in descending order
+        channels.sort_by(|a, b| b.created_at.cmp(&a.created_at));
+        Ok(ListChannelsResult { channels })
```

Comment:
Can we also update https://github.com/nervosnetwork/cfn-node/blob/1297afde388981ef14c1e498cf34e4ad1baef7d0/src/rpc/channel.rs#L78 to include the `created_at` field?

Reply:
it's done in the PR?

Reply:
Oh, sorry. I overlooked that. Can you also give an example of the output format of the DateTime<Utc>? Since this will be shown to the users. I think we need to make this unambiguous and easily readable. I don't know if the default serde rule is good enough.

Reply:
looks like this:
```console
created_at: '2024-06-29T15:14:44.765077Z'
```

Reply:
So, this is ISO 8601. I think we'd better make this serialization rule more explicit lest upstream changes break down stream applications. Some documentation for the date time format will also be useful to downstream application developers. Since we don't have rpc documentation yet. We can just add a Todo in the code.

Reply:
> looks like this:
> 
> ```
> created_at: '2024-06-29T15:14:44.765077Z'
> ```

please follow ckb's rpc convention, use the epoch timestamp in rpc field and displayed as hex string: https://github.com/nervosnetwork/ckb/tree/develop/rpc#type-timestamp